packages/pool/contracts/StateUtils.sol

```diff
@@ -519,14 +534,6 @@ contract StateUtils is IStateUtils {
         }
         else
         {
-            if (addressCheckpointArray.length > MAX_INTERACTION_FREQUENCY)
-            {
-                uint256 interactionTimestampMaxInteractionFrequencyAgo = snapshotBlockToTimestamp[addressCheckpointArray[addressCheckpointArray.length - MAX_INTERACTION_FREQUENCY].fromBlock];
-                require(
```

Comment:
We don't need to throttle additions to users.delegates because there is already the one delegation update per EPOCH_LENGTH limit